src/ng/cacheFactory.js

```diff
@@ -223,6 +223,8 @@ function $CacheFactoryProvider() {
             delete lruHash[key];
           }
 
+          if (!(key in data)) return;
```

Comment:
Is that literally what this line should be? Sorry, this is my first time contributing here and I'm having trouble finding anything about a `createMap()` function. Is this just pseudo-code, or something internal and already available?